portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -14,35 +14,99 @@
 
 package com.google.sps.servlets;
 
+import java.util.List;
 import java.io.IOException;
 import java.util.ArrayList;
 import com.google.gson.Gson;
-import javax.servlet.annotation.WebServlet;
+import com.google.sps.data.Comment;
 import javax.servlet.http.HttpServlet;
+import javax.servlet.annotation.WebServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
+import com.google.appengine.api.datastore.Query;
+import com.google.appengine.api.datastore.Entity;
+import com.google.appengine.api.datastore.FetchOptions;
+import com.google.appengine.api.datastore.PreparedQuery;
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.Query.SortDirection;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
 
 
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
 
-    String convertTojson(ArrayList<String> comments) {
-        Gson gson = new Gson();
-        String convertedTojson = gson.toJson(comments);
-        return convertedTojson;
-    }
+  //this is the number of comments that should
```

Comment:
In general comments should have a space after the comment signifier (eg:

`// This is the number....`

Also capitalize / proper punctuation on any comments that are a sentance.